src/modules/studentLife/pages/StudentLifePage/StudentLife.test.tsx

```diff
@@ -0,0 +1,10 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import { StudentLifePage } from './StudentLifePage.component';
+
+describe('[Component] StudentLifePage', () => {
+  it('renders correctly', () => {
+    const studentLifePage = render(<StudentLifePage />);
+    expect(studentLifePage).toMatchSnapshot();
```

Comment:
you did not run your tests, you are missing a snapshot here :) run `yarn test` to run all tests

Reply:
![image](https://user-images.githubusercontent.com/62103600/93817591-7a28bb00-fc59-11ea-9ce6-431c9a2ed313.png)

wtf C:\Users\Rinkichi\Desktop\etuutt-front\src\index.tsx
   1:27  error  Delete `␍`  prettier/prettier
   2:34  error  Delete `␍`  prettier/prettier
   3:23  error  Delete `␍`  prettier/prettier
   4:62  error  Delete `␍`  prettier/prettier
   5:60  error  Delete `␍`  prettier/prettier
   6:1   error  Delete `␍`  prettier/prettier
   7:17  error  Delete `␍`  prettier/prettier
   8:21  error  Delete `␍`  prettier/prettier
   9:22  error  Delete `␍`  prettier/prettier
  10:23  error  Delete `␍`  prettier/prettier
  11:34  error  Delete `␍`  prettier/prettier
  12:3   error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\internationalization\service\i18n.service.ts
   1:28  error  Delete `␍`  prettier/prettier
   2:65  error  Delete `␍`  prettier/prettier
   3:50  error  Delete `␍`  prettier/prettier
   4:52  error  Delete `␍`  prettier/prettier
   5:1   error  Delete `␍`  prettier/prettier
   6:51  error  Delete `␍`  prettier/prettier
   7:64  error  Delete `␍`  prettier/prettier
   8:30  error  Delete `␍`  prettier/prettier
   9:55  error  Delete `␍`  prettier/prettier
  10:1   error  Delete `␍`  prettier/prettier
  11:22  error  Delete `␍`  prettier/prettier
  12:8   error  Delete `␍`  prettier/prettier
  13:1   error  Delete `␍`  prettier/prettier
  14:56  error  Delete `␍`  prettier/prettier
  15:13  error  Delete `␍`  prettier/prettier
  16:21  error  Delete `␍`  prettier/prettier
  17:23  error  Delete `␍`  prettier/prettier
  18:4   error  Delete `␍`  prettier/prettier
  19:1   error  Delete `␍`  prettier/prettier
  20:21  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\pages\index.ts
  1:33  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\services\index.ts
  1:35  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\modules\navigation\services\routes.services.test.ts
   1:60  error  Delete `␍`  prettier/prettier
   2:1   error  Delete `␍`  prettier/prettier
   3:45  error  Delete `␍`  prettier/prettier
   4:47  error  Delete `␍`  prettier/prettier
   5:27  error  Delete `␍`  prettier/prettier
   6:54  error  Delete `␍`  prettier/prettier
   7:1   error  Delete `␍`  prettier/prettier
   8:32  error  Delete `␍`  prettier/prettier
   9:6   error  Delete `␍`  prettier/prettier
  10:1   error  Delete `␍`  prettier/prettier
  11:53  error  Delete `␍`  prettier/prettier
  12:31  error  Delete `␍`  prettier/prettier
  13:63  error  Delete `␍`  prettier/prettier
  14:1   error  Delete `␍`  prettier/prettier
  15:36  error  Delete `␍`  prettier/prettier
  16:6   error  Delete `␍`  prettier/prettier
  17:1   error  Delete `␍`  prettier/prettier
  18:54  error  Delete `␍`  prettier/prettier
  19:44  error  Delete `␍`  prettier/prettier
  20:76  error  Delete `␍`  prettier/prettier
  21:1   error  Delete `␍`  prettier/prettier
  22:43  error  Delete `␍`  prettier/prettier
  23:6   error  Delete `␍`  prettier/prettier
  24:4   error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\react-app-env.d.ts
  1:40  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\setupTests.ts
  1:66  error  Delete `␍`  prettier/prettier
  2:33  error  Delete `␍`  prettier/prettier
  3:47  error  Delete `␍`  prettier/prettier
  4:59  error  Delete `␍`  prettier/prettier
  5:50  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\translations\locale\index.ts
  1:28  error  Delete `␍`  prettier/prettier
  2:28  error  Delete `␍`  prettier/prettier
  3:1   error  Delete `␍`  prettier/prettier
  4:27  error  Delete `␍`  prettier/prettier

C:\Users\Rinkichi\Desktop\etuutt-front\src\__tests__\App.test.tsx
  1:49  error  Delete `␍`  prettier/prettier
  2:27  error  Delete `␍`  prettier/prettier
  3:30  error  Delete `␍`  prettier/prettier
  4:1   error  Delete `␍`  prettier/prettier
  5:36  error  Delete `␍`  prettier/prettier
  6:31  error  Delete `␍`  prettier/prettier
  7:33  error  Delete `␍`  prettier/prettier
  8:4   error  Delete `␍`  prettier/prettier

✖ 76 problems (76 errors, 0 warnings)
  76 errors and 0 warnings potentially fixable with the `--fix` option.